scripts/private-archive-preprocess/asiaq_private_placenames/README.md

```diff
@@ -0,0 +1,24 @@
+# ASIAQ placenames preprocessing steps
+
+## 20201112_Oqaasileriffik_place-name register.zip
+
+Provided by Eva Mätzler. This data was placed in the appropriate private archive
+location and then unzipped:
+
+```
+cd /share/appdata/qgreenland-private-archive/asiaq_private_placenames/
+unzip 20201112_Oqaasileriffik_place-name register.zip
+```
+
+
+## 2021 0119_Oqaasileriffik_place-name categories_ENG.xlsx
+
+Provided by Arnaq B. Johansen via email on Jan 19, 2021.
+
+This file was placed in the appropriate private archive location and then
+manually converted to a .csv using libreoffice.
```

Comment:
What is the `.csv` filename? (I'm assuming same, just different extension; worth it to be explicit here?